client/keys/utils.go

```diff
@@ -31,8 +32,8 @@ type KeyOutput struct {
 // GetKeyBase initializes a keybase based on the configuration
 func GetKeyBase() (keys.Keybase, error) {
 	if keybase == nil {
-		rootDir := viper.GetString(cli.HomeFlag)
-		db, err := dbm.NewGoLevelDB(KeyDBName, rootDir)
+		rootDir := filepath.Join(viper.GetString(cli.HomeFlag), ".tlc")
```

Comment:
.tlc?

Reply:
Per https://github.com/cosmos/cosmos-sdk/blob/develop/client/keys/README.md

"Tendermint Light Client", maybe?

Can be changed

Reply:
oh duh.

the resulting path is a bit weird: `ls ~/.basecli/.tlc/data/keys.db/`

Maybe `~/.basecli/keys/keys.db` would be nicer